lib/components/Editor/CustomExtensions/KeyboardShortcuts/helpers.js

```diff
@@ -0,0 +1,12 @@
+export const generateKeyboardShortcuts = (editor, shortcuts) => {
+  return shortcuts.reduce(
+    (keyboardShortcuts, shortcut) => ({
+      ...keyboardShortcuts,
+      [shortcut.key]: () => {
+        shortcut.handler?.(editor.getHTML(), editor);
```

Comment:
IMO, we need to pass only `editor` as argument to the `handler` function and users can choose to whether call `getHTML` or not as per their requirement to avoid unnecessary overhead. Is there any other use case for passing the html as well? please share your thoughts